### ISSUE TITLE
Played game no longer displayed?

### ISSUE THREAD

**Issue by danzotron** [EionRobb/pidgin-opensteamworks#165]

**Comment by EionRobb** [comment]
Wow yeah, looks like they've changed how they send through in-game notifications.  Now instead of sending it along with other events, it's a separate lookup to https://steamcommunity.com/chat/friendstate/25340362 needed when there's a "personastate" event with "status_flags": 9055,"persona_state": 1,

**Comment by danzotron** [comment]
Its not a big deal just thought I'd mention it. Everything else is just fine.

Thanks for developing this plugin by the way, it has served us well. I tried to look for a btc donation address or paypal etc but couldn't seem to find anything.

**Comment by EionRobb** [comment]
@danzotron can you try to compile from git?  I think I've fixed it in https://github.com/EionRobb/pidgin-opensteamworks/commit/7f761df2746a8ab518fd620e50df089421df9819 but would appreciate a confirmation :)

**Comment by Vitagox** [comment]
I know I didn't open the thread, but a built dll would be wonderful, not seeing the games appear make my brain itch and I was wondering if it was Pidgin or the plugin for the longest time.

I also want to offer my gratitude for this plugin, it's nice to have an alternative from needing to have a browser window open if I just want to chat.

**Comment by EionRobb** [comment]
Sure. Will post a link to the dll shortly. 

It's been working great for me, so will probably post a new version release shortlyish too

**Comment by EionRobb** [comment]
Whoops, forgot to update here with the url to the latest code dll thing

https://eion.robbmob.com/libsteam-debug.dll

**Comment by gonzoid** [comment]
Hello @EionRobb, I just noticed that the Windows download link on your **SkypeWeb** Plugin page is broken :

- it currently points to https://eion.robbmob.com/libskypeweb-debug.dll
- instead of https://eion.robbmob.com/libskypeweb.dll

If you want, I can create an issue on [EionRobb/skype4pidgin](https://github.com/EionRobb/skype4pidgin).

**Comment by phobeus81** [comment]
I also just noticed that I am affected by this bug using 1.6.1 (Fedora) and wondering why recently nobody is playing this year anymore and eagerly involved in work ;)

I tried the recent version from git that seems to contain 7f761df and tried to build it on my system, replacing the system .so with the one from the build: cp libsteam.so /usr/lib64/purple-2/libsteam.so.

After a restart of pidgin, it does still not show the icon when people are in-game. So if I didn't do anything wrong, this seems not to fix the issue.

**Comment by EionRobb** [comment]
@phobeus81 double-check if you're not loading in another version of the .so too, (Open up Help->Debug Window, leave that open then open Tools->Plugins, close the plugins window and pause the debug window output, and look for "plugins: probing .... libsteam")

**Comment by phobeus81** [comment]
yeah, sorry, I think it was a false claim. Looks like it didn't update on people who had been already in game. I was assuming that on startup it loads the state. Someone who went into game after applying the patch is shown perfectly fine. So seems like it fixes the issue.

**Comment by EionRobb** [comment]
@phobeus81 sounds like it isn't really fixed. :) 

I'll have a look into how we can get the initial state too
